namespace TestWork.Tasks.Domain.Exceptions;

public class TaskNotFoundException() : Exception("Не найдена задача по идентификатору");